response_operations_ui/controllers/survey_controllers.py

```diff
@@ -276,7 +276,7 @@ def create_survey(survey_ref, short_name, long_name, legal_basis, survey_mode):
             {"name": "COLLECTION_INSTRUMENT", "classifierTypes": ["FORM_TYPE"]},
             {"name": "COMMUNICATION_TEMPLATE", "classifierTypes": ["LEGAL_BASIS", "REGION"]},
         ],
-        "eqVersion": "v2" if survey_mode != "SEFT" else "",
+        "eqVersion": "v3" if survey_mode != "SEFT" else "",
```

Comment:
Strictly speaking not part of this PR and in the future will be redundant, however new surveys should always be v3 not v2 now

Reply:
Good job catching this.  Could've lead to some interesting issues if we hadn't done this and the similar collection exercise change below >_<